astropy/convolution/convolve.py

```diff
@@ -653,13 +663,19 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
         if fft_pad is None:
             # default is 'True' according to the docstring
             fft_pad = True
+        if dealias is None:
+            # default is 'False' according to the docstring
+            dealias = False
     elif boundary == 'wrap':
         if psf_pad:
             raise ValueError("With boundary='wrap', psf_pad cannot be enabled.")
         psf_pad = False
         if fft_pad:
             raise ValueError("With boundary='wrap', fft_pad cannot be enabled.")
         fft_pad = False
+        if fft_pad:
```

Comment:
Did you intend to check for `dealias` here?

Reply:
Fixed in cfaa006.